languages/csharp/exercises/concept/nullability/.meta/Example.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+
+public static class Nullability
+{
+
+    public static int ComputeWidthPx(string? FirstName,
```

Comment:
I really, really like how you've managed to use both a nullable reference type (strings) and a nullable value type (integer), and it makes perfect sense. My only concern is that there is just a single method for the student to work with, which means the student has to get it right completely. Do you think it would be possible to split this method into several methods? One option could be to have a separate method for just composing the name.

Reply:
Yes I see, maybe one method to compute the width of a given string and a method to compute assemble the parts.
I will work on that.